pyro/contrib/epidemiology/sir.py

```diff
@@ -479,16 +479,30 @@ def predict(self, forecast=0):
 
 
 class RegionalSIRModel(CompartmentalModel):
-    """
+    r"""
     Susceptible-Infected-Recovered model with coupling across regions.
 
     To customize this model we recommend forking and editing this class.
 
     This is a stochastic discrete-time discrete-state model with three
-    compartments: "S" for susceptible, "I" for infected, and "R" for
-    recovered individuals (the recovered individuals are implicit: ``R =
+    compartments in each region: "S" for susceptible, "I" for infected, and "R"
+    for recovered individuals (the recovered individuals are implicit: ``R =
     population - S - I``) with transitions ``S -> I -> R``.
 
+    Regions are coupled by a ``coupling`` matrix with entries in ``[0,1]``.
+    The all ones matrix is equivalent to a single region. The identity matrix
+    is equivalent to a set of independent regions. This need not be symmetric,
+    but symmetric matrices are probably more physically plausible. The expected
+    number of new infections each time step ``S2I`` is Binomial distributed
+    with mean::
+
+        E[S2I] = S (1 - (1 - R0 / (population @ coupling)) ** (I @ coupling))
+               ≈ R0 S (I @ coupling) / (population @ coupling)  # for small I
```

Comment:
will sphinx be happy with /approx?

Reply:
yes sphinx has been happily rendering our unicode for a few months now.